data/input_files/default.lua

```diff
@@ -4,6 +4,13 @@ dt      = 0.25
 
 output_type = "SidreVisIt"
 
+-- types = l2, full, average, min, max
+-- frequency of dump vs end value
+output_fields_json = {
+    [0] = {"temperature", "l2"},
+    [1] = {"displacement", "full"}
+}
+
```

Comment:
Was this something you were testing for #465?

Reply:
oh that definitely didnt mean to be checked in... ill revert